app/scripts/views/sign_up.js

```diff
@@ -83,7 +84,7 @@ define(function (require, exports, module) {
       return coppaView.render()
         .then(() => {
           this.trackChildView(coppaView);
-          coppaView.on('submit', this.validateAndSubmit.bind(this));
+          coppaView.on('submit', () => this.trigger('submit'));
```

Comment:
Would an explanatory comment in the code help?

I added it to fix the problem of pressing `ENTER` from the coppa input not triggering a `submit` event in the parent form. I need that event to hang my flow event on.